test/mitmproxy/contentviews/test_json.py

```diff
@@ -13,6 +13,7 @@ def test_parse_json():
         b'{"foo" : "\xe4\xb8\x96\xe7\x95\x8c"}'
     )  # utf8 with chinese characters
     assert json.parse_json(b'{"foo" : "\xFF"}') is json.PARSE_ERROR
+    assert json.parse_json(b'{"foo": []}') is not json.PARSE_ERROR
```

Comment:
This specific test method is testing for parsing errors. Is this additional test case here adding any value to that? I feel it's not, so let's remove it. The additions below are all good and cover it sufficiently!